tests/vu/lower/efu.expected

```diff
@@ -0,0 +1,184 @@
+  ELENG CVF_GARBAGE1: 00000000/+0.00
+  ELENG CVF_GARBAGE2: 5f1677e2/+10842383117545635840.00
+  ELENG CVF_INCREASING: 406f7750/+3.74
+  ELENG CVF_DECREASING: 40ac5345/+5.39
+ERCPR:
+  ERCPR CVF_ZERO: 7fffffff/+NaN
+  ERCPR CVF_NEGZERO: ffffffff/-NaN
+  ERCPR CVF_MAX: 00000000/+0.00
+  ERCPR CVF_MIN: 80000000/-0.00
+  ERCPR CVF_MAX_MANTISSA: 3f000001/+0.50
+  ERCPR CVF_MAX_EXP: 00000000/+0.00
+  ERCPR CVF_MIN_EXP: 7fffffff/+NaN
+  ERCPR CVF_ONE: 3f800000/+1.00
+  ERCPR CVF_NEGONE: bf800000/-1.00
+  ERCPR CVF_GARBAGE1: 7fffffff/+NaN
+  ERCPR CVF_GARBAGE2: a03c98eb/-0.00
+  ERCPR CVF_INCREASING: 3eaaaaab/+0.33
+  ERCPR CVF_DECREASING: 3f000000/+0.50
+ERLENG:
+  ERLENG CVF_ZERO: 7fffffff/+NaN
+  ERLENG CVF_NEGZERO: 7fffffff/+NaN
+  ERLENG CVF_MAX: 1f3504f3/+0.00
+  ERLENG CVF_MIN: 1f3504f3/+0.00
+  ERLENG CVF_MAX_MANTISSA: 3e93cd3b/+0.29
+  ERLENG CVF_MAX_EXP: 1f3504f3/+0.00
+  ERLENG CVF_MIN_EXP: 7fffffff/+NaN
+  ERLENG CVF_ONE: 3f13cd3a/+0.58
+  ERLENG CVF_NEGONE: 3f13cd3a/+0.58
+  ERLENG CVF_GARBAGE1: 7fffffff/+NaN
+  ERLENG CVF_GARBAGE2: 1fd9c601/+0.00
+  ERLENG CVF_INCREASING: 3e88d677/+0.27
+  ERLENG CVF_DECREASING: 3e3e26eb/+0.19
+ERSADD:
+  ERSADD CVF_ZERO: 7fffffff/+NaN
+  ERSADD CVF_NEGZERO: 7fffffff/+NaN
+  ERSADD CVF_MAX: 00000000/+0.00
+  ERSADD CVF_MIN: 00000000/+0.00
+  ERSADD CVF_MAX_MANTISSA: 3daaaaac/+0.08
+  ERSADD CVF_MAX_EXP: 00000000/+0.00
+  ERSADD CVF_MIN_EXP: 7fffffff/+NaN
+  ERSADD CVF_ONE: 3eaaaaab/+0.33
+  ERSADD CVF_NEGONE: 3eaaaaab/+0.33
+  ERSADD CVF_GARBAGE1: 7fffffff/+NaN
+  ERSADD CVF_GARBAGE2: 00000000/+0.00
+  ERSADD CVF_INCREASING: 3d924925/+0.07
+  ERSADD CVF_DECREASING: 3d0d3dcb/+0.03
+ERSQRT:
+  ERSQRT CVF_ZERO: 7fffffff/+NaN
+  ERSQRT CVF_NEGZERO: 7fffffff/+NaN
+  ERSQRT CVF_MAX: 1f3504f3/+0.00
+  ERSQRT CVF_MIN: 1f3504f3/+0.00
+  ERSQRT CVF_MAX_MANTISSA: 3f3504f3/+0.71
+  ERSQRT CVF_MAX_EXP: 1f800000/+0.00
+  ERSQRT CVF_MIN_EXP: 7fffffff/+NaN
+  ERSQRT CVF_ONE: 3f800000/+1.00
+  ERSQRT CVF_NEGONE: 3f800000/+1.00
+  ERSQRT CVF_GARBAGE1: 7fffffff/+NaN
+  ERSQRT CVF_GARBAGE2: 2fdbbaac/+0.00
+  ERSQRT CVF_INCREASING: 3f13cd3a/+0.58
+  ERSQRT CVF_DECREASING: 3f3504f3/+0.71
+ESADD:
+  ESADD CVF_ZERO: 00000000/+0.00
+  ESADD CVF_NEGZERO: 00000000/+0.00
+  ESADD CVF_MAX: 7fffffff/+NaN
+  ESADD CVF_MIN: 7fffffff/+NaN
+  ESADD CVF_MAX_MANTISSA: 413ffffe/+12.00
+  ESADD CVF_MAX_EXP: 7fffffff/+NaN
+  ESADD CVF_MIN_EXP: 00000000/+0.00
+  ESADD CVF_ONE: 40400000/+3.00
+  ESADD CVF_NEGONE: 40400000/+3.00
+  ESADD CVF_GARBAGE1: 00000000/+0.00
+  ESADD CVF_GARBAGE2: 7eb0e16b/+117557282134571560288112383319651385344.00
+  ESADD CVF_INCREASING: 41600000/+14.00
+  ESADD CVF_DECREASING: 41e80000/+29.00
+ESIN:
+  ESIN CVF_ZERO: 00000000/+0.00
+  ESIN CVF_NEGZERO: 00000000/+0.00
+  ESIN CVF_MAX: 7fd76aa2/+NaN
+  ESIN CVF_MIN: ffd76aa2/-NaN
+  ESIN CVF_MAX_MANTISSA: 3f68c816/+0.91
+  ESIN CVF_MAX_EXP: 7f2ed54b/+232393151678163554045421220250750287872.00
+  ESIN CVF_MIN_EXP: 00000000/+0.00
+  ESIN CVF_ONE: 3f576aa4/+0.84
```

Comment:
Yeah, FF3 required more accuracy already for values outside [0, 4) in our case, so we applied a modulus - that's why we didn't need -1 (-0.5pi), because that's actually for us.

Makes sense that this wouldn't operate the same.  Hope no games are crazy enough to rely on the bits like in that one example you gave from Tri-Ace.

-[Unknown]